reana_commons/k8s/secrets.py

```diff
@@ -148,6 +149,49 @@ def get_secrets(self):
 
         return secrets_with_types
 
+    def get_env_secrets_specs(self):
+        """Get a list of specification items for env type secrets for k8s."""
```

Comment:
What do you think about the following changes?

The function renamed to: `get_env_secrets_as_k8s_spec` or similar. Since we are attaching them to Kubernetes objects in RWC and RJC I thought it could make sense to reference Kubernetes in the name.

About the docstring: `Return all environment variable secrets as a list of Kubernetes environment variable specs.` Reference https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1EnvVar.md.

Reply:
We just discussed with @diegodelemos about doctoring. What do you think about `Return all secrets of type env as a list of Kubernetes environment variable specs.` ?